projects/golang/go/development/docs/eks_go_patching_runbook.md

```diff
@@ -0,0 +1,145 @@
+# EKS Go Developer Documentation
+
+Documentation for EKS Go developers and those looking to extend and work with the EKS Go build and distribution systems.
+
+## Contents
+1. [Adding a Go minor version to EKS Go](#adding-a-go-minor-version-to-eks-go)
+1. [Updating an EKS Go minor version to use a new upstream Go patch version](#new-patch-versions)
+
+## Adding a Go Minor Version to EKS Go
+When a new Golang minor version is released EKS Go will need to track and build that version.
+This guide lays out the steps to add a new Go minor version to EKS Go.
+
+This work breaks down into a few steps:
+
+1. [setup the project structure in EKS Distro Build Tooling](#setup-project-structure)
+1. [setup the Golang RPM spec in EKS Distro Build Tooling](#golang-rpm-spec)
+1. [configure EKS Go prow pre-submits in EKS Distro Prow Jobs](#prow-jobs-pre-submits)
+1. [configure EKS Go prow post-submits in EKS Distro Prow Jobs](#prow-jobs-post-submits)
+
+### Setup Project Structure
+
+First, we need to set up the new Go minor version project structure, mirroring the other Go minor versions supported by EKS Go.
+You can use the [the helper script](../scripts/setup_golang_minor_version.sh) to set up the initial structure.
+For example, to set up Go `1.19` starting at the Git tag `go1.19.2`, 
+you would invoke the following command in a clone of EKS Distro Build Tooling and commit the results:
```

Comment:
Fork?

Reply:
yeah, we can specify that this should be done on the developers fork of EKS D build tooling

Reply:
alrighty updated, and include forking eksdbt as a step 0